src/components/modals/InfoModal.tsx

```diff
@@ -18,54 +18,91 @@ export const InfoModal = ({ isOpen, handleClose }: Props) => {
           isRevealing={true}
           isCompleted={true}
-          value="L"
+          value="O"
           status="present"
         />
-        <Cell value="O" isCompleted={true} />
-        <Cell value="T" isCompleted={true} />
+        <Cell value="S" isCompleted={true} />
+        <Cell value="4" isCompleted={true} />
+        <Cell value=":" isCompleted={true} />
+        <Cell value="0" isCompleted={true} />
+        <Cell value="9" isCompleted={true} />
       </div>
       <p className="text-sm text-gray-500 dark:text-gray-300">
-        The letter L is in the word but in the wrong spot.
+        The letter O is in the word but in the wrong spot.
       </p>
 
       <div className="mb-1 mt-4 flex justify-center">
-        <Cell value="V" isCompleted={true} />
-        <Cell value="A" isCompleted={true} />
-        <Cell value="G" isCompleted={true} />
-        <Cell isRevealing={true} isCompleted={true} value="U" status="absent" />
+        <Cell value="L" isCompleted={true} />
+        <Cell value="U" isCompleted={true} />
+        <Cell isRevealing={true} isCompleted={true} value="K" status="absent" />
         <Cell value="E" isCompleted={true} />
+        <Cell value="9" isCompleted={true} />
+        <Cell value=":" isCompleted={true} />
+        <Cell value="2" isCompleted={true} />
+        <Cell value="1" isCompleted={true} />
       </div>
       <p className="text-sm text-gray-500 dark:text-gray-300">
-        The letter U is not in the word in any spot.
+        The letter K is not in the word.
       </p>
-
-      <p className="mt-6 text-sm italic text-gray-500 dark:text-gray-300">
-        This is an open source version of the word guessing game we all know and
-        love -{' '}
-        <a
-          href="https://github.com/cwackerfuss/react-wordle"
-          className="font-bold underline"
-        >
-          check out the code here
-        </a>{' '}
+      <div className="mb-1 mt-4 flex justify-center">
+        <Cell value="J" isCompleted={true} />
+        <Cell value="O" isCompleted={true} />
+        <Cell value="B" isCompleted={true} />
+        <Cell value="2" isCompleted={true} status="incorrectCharType" />
+        <Cell value="3" isCompleted={true} />
+        <Cell value=":" isCompleted={true} />
+        <Cell value="1" isCompleted={true} />
+        <Cell value="6" isCompleted={true} />
+      </div>
+      <p className="text-sm text-gray-500 dark:text-gray-300">
+        The number 2 is an incorrect character type, try a number or colon.
+      </p>
+      <div className="mb-1 mt-4 flex justify-center">
+        <Cell value="P" isCompleted={true} />
+        <Cell value="S" isCompleted={true} />
+        <Cell value="A" isCompleted={true} />
+        <Cell value="L" isCompleted={true} />
+        <Cell value="M" isCompleted={true} />
+        <Cell value="1" isCompleted={true} />
+        <Cell value="1" isCompleted={true} status="low" />
+        <Cell value="6" isCompleted={true} />
```

Comment:
"Psalm116" is not a valid guess. It would not pass the RegEx validation as Psalms has more than 1 chapter.